NuGet.config

```diff
@@ -28,6 +29,7 @@
     <clear />
     <!--Begin: Package sources managed by Dependency Flow automation. Do not edit the sources below.-->
     <!--  Begin: Package sources from dotnet-corefx -->
+    <add key="darc-int-dotnet-corefx-6e37696" value="true" />
```

Comment:
This should be removed 

cc @wtgodbe

Reply:
These can actually stay now - the section at the bottom of the file disables these in non-internal builds